ASTProcessor/build.gradle.kts

```diff
@@ -0,0 +1,72 @@
+plugins {
+	application
+	alias(libs.plugins.springboot.web) apply true
+	alias(libs.plugins.dependency.management) apply true
+	id("org.sonarqube") version "5.1.0.4882"
```

Comment:
You can put the dependencies inside the `libs.versions.toml` file, you can use this repository to guide you on how the libraries are inside the file:

[ArgosInstrumentation-libs.versions.toml](https://github.com/SoftDevSix/ArgosInstrumentation/blob/develop/gradle/libs.versions.toml)

Reply:
I think this change is unnecessary considering this implementation going to move to the principal project and  the subproject going to be removed

Reply:
Even if the subproject is deleted, the `libs.versions.toml` file must have all the versions of the dependencies so that it does not have to be modified throughout the file and only 1 file has to be modified, this to have more consistency. Even if it were taken to a separate project, the project would have to have a `libs.versions.toml` file for the versions.

Reply:
Done ✅